### PR TITLE
chromium: Post-ozone-wayland cleanups

### PR DESCRIPTION
**Pull request by rakuco**
This PR includes several clean-ups after the ozone-wayland-related pull requests were merged a few weeks ago.

They mainly reduce duplication and drop a few `.inc` files to reduce confusion.

### CODE REVIEW

**Comment by rakuco** [comment]
@otavio @msisov

**Comment by otavio** [comment]
The jumbo build could be configured somehow, don't you think? If so, what we default?

**Comment by rakuco** [comment]
> The jumbo build could be configured somehow, don't you think? If so, what we default?

It can; one just needs to append `use_jumbo_build=true` to `GN_ARGS` locally. By default, I'd say we follow upstream (which sets it to false).

File: recipes-browser/chromium/chromium-ozone-wayland_64.0.3274.0.r517731.igalia.1.bb [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+        libxkbcommon \
+        virtual/egl \
+        wayland \
+"
+
+GN_ARGS += "\
+        enable_mus=true \
+        use_ozone=true \
+        ozone_auto_platforms=false \
+        ozone_platform_headless=true \
```
Comment (msisov) [inline recipes-browser/chromium/chromium-ozone-wayland_64.0.3274.0.r517731.igalia.1.bb]: can you set it false as well? I believe it's not needed.

Comment (rakuco) [reply recipes-browser/chromium/chromium-ozone-wayland_64.0.3274.0.r517731.igalia.1.bb]: Would you be OK with doing that in a separate PR? Changing the value of the option causes the build to change, whereas I was hoping the cleanups here would have no effect on the actual build.

Comment (msisov) [reply recipes-browser/chromium/chromium-ozone-wayland_64.0.3274.0.r517731.igalia.1.bb]: please go ahead as you wish!

Comment (rakuco) [reply recipes-browser/chromium/chromium-ozone-wayland_64.0.3274.0.r517731.igalia.1.bb]: ... though arguably removing `use_jumbo_build=true` does change how things are built... I'll send that part in a separate PR too.

**Comment by rakuco** [comment]
@msisov @otavio I've pushed a new version that doesn't change the Jumbo builds part. There shouldn't be anything controversial now.

**Pull request merged**